data/deep missions.txt

```diff
@@ -0,0 +1,1366 @@
+
+
+
+# Lore about the mysterious cubes:
+# These cubes, called sensor cubes, are used by the Deep to track fleet movements above the planet which they are placed on, most importantly the movements of pirate fleets.
+# The Deep uses merchants to place the cubes to try and make them more inconspicuous
+# Information gathered from these cubes is used by Deep Security and is on rare occasions handed over to the Republic or Navy (DS is an independent police force and therefore is under no jurisdiction to have to hand over all its info to the Navy).
+# For some reason, pirates know of this and send bounty hunters after anyone they know are in possesion of these cubes, stealing them before destroying the merchant.
+# Cubes are asked to be retrieved by merchants for reasons unknown; malfunctions? just old sensors? One can only wonder.
+
+# Deep: Mystery Cubes missions will involve first... I have no idea.
+# I'm thinking about having one of the missions involve going to Carbuncle Station
+# Maybe something to do with assisting Deep Security in tracking down a warlord (DS would still be low on ships during this time for whatever reason)
+# Might have to get the Navy involved
+mission "Deep: Mystery Cubes"
+	minor
+	name ``
+	description ``
+	source
+		attributes deep
+	destination
+	to offer
+		"random" < 40
+		"deep convoy" >= 1
+	on offer
+		conversation
+			``
+
+
+
+mission "Deep: TMBR 0"
+	minor
+	name `Travel to <planet>`
+	description `Travel to <destination> where you last left the band "There Might Be Riots."`
+	source
+		near Naos 1 100
+		attributes deep
+	destination Allhome
+	to offer
+		random < 60
+		"Transport scientists: done" >= 3
+		"Escort science vessel: done" >= 3
+		"Record academic conference: done" >= 3
+		has "There Might Be Riots part 3B: done"
+		not "Deep: Remnant 0"
+		not "Deep: Singularity"
+	on offer
+		conversation
+			`When you enter the <planet> spaceport, you hear what is undoubtedly a "There Might Be Riots" song playing loud enough for anyone in the spaceport to hear. You look around and find that the source of the music is the external speakers of a Bounder that is surrounded by a few people dancing.`
+			choice
+				`	(Dance with them.)`
+				`	(Ignore them.)`
+					defer
+					
+			`	You grow nearer to the Bounder and count five people dancing in total: three women and two men. One of the woman dancing notices you walking over. "It looks like someone else is about to join the party!" This is met by a collective cheer from the rest of the dancers.`
+			`	A few minutes of dancing later, one of the people introduces himself to you as Garrison. He lists off the names of the rest of the people with him; Isaac, Maria, Hannah, and Larua, then explains that they are all Deep scientists. They are currently enjoying a short vacation listening to their favorite band before they must travel to Midgard to continue their work for the Deep.`
+			`	"How do you know about 'There Might Be Riots?'" Hannah asks you.`
```

Comment:
Lol, who's Elton John?

Jokes aside, whether TMBR are popular enough or not, Hannah assumes you know the artist of the songs that are playing in the first place. The player could have just been some rando who wanted to join in on a party, so how the scientists are lead to knowing that the player knows TMBR should be changed in general.